app/models/queued_email.rb

```diff
@@ -223,30 +229,24 @@ def send_email
     return true unless RunLevel.is_normal?
 
     log_msg = "SEND #{flavor} " \
-      "from=#{begin
-                user.login
-              rescue StandardError
-                "nil"
-              end} " \
-      "to=#{begin
-              to_user.login
-            rescue StandardError
-              "nil"
-            end} " +
-              queued_email_integers.map { |x| "#{x.key}=#{x.value}" }.
-              join(" ") +
-              queued_email_strings.map { |x| "#{x.key}=\"#{x.value}\"" }.
-              join(" ")
+      "from=#{user&.login || "nil"} " \
+      "to=#{to_user&.login || "nil"} " +
+              queued_email_integers.map { |x|
+                "#{x.key}=#{x.value}"
+              }.join(" ") +
+              queued_email_strings.map do |x|
+                "#{x.key}=\"#{x.value}\""
+              end.join(" ")
```

Comment:
@pellaea - Here are those statements without the `begin...rescue...end`. Sanity check that this is correct...

Reply:
That looks right to me.  Hopefully it won't break anything! :)  I think the only way user.login would ever have failed is if user was nil, in which case this should be equivalent.  And user.login will always be non-falsey, so again should be safe.